compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIECreateLogicalObjectFifoLink.cpp

```diff
@@ -88,15 +88,11 @@ LogicalResult createLogicalObjectFifoLink(
       auto sourceLogicalObjectFifo =
           dyn_cast<AMDAIE::LogicalObjectFifoFromMemrefOp>(
               stridedOp.getSource().getDefiningOp());
-      if (!sourceLogicalObjectFifo) {
-        stridedOp->emitError(
-            "does not have a `LogicalObjectFifoFromMemrefOp` as source");
-        return failure();
-      }
       if (!lastUserOp || lastUserOp->isBeforeInBlock(stridedOp)) {
         lastUserOp = stridedOp;
       }
-      if (logicalObjectFifo == sourceLogicalObjectFifo) {
+      if (sourceLogicalObjectFifo &&
```

Comment:
Just add a comment here that it can either be a `from_memref` or a `placeholder`.

Reply:
Added it.